README.md

```diff
@@ -1,20 +1,157 @@
+
+
+
+# <a name="target"></a>
+# Target Audience
+
+
+
+1. Machine Learning Fairness Researchers: ML researchers can easily add their unfairness mitigation techniques to this repository and use the assessment toolkit to load their original and refined model predictions to get insights into how much they have improved the disparity metrics of interest.
+
+
+2. Developers/ ML Engineers/ Data Scientists: Having a variety of curated unfairness mitigations techniques in one place makes it easier for data scientists and (ML) developers to experiment with different unfairness techniques in a seamless manner. The set of rich interactive visualizations allow developers and data scientists to easily visualize the unfairness happening in their machine learning models instead of wasting time and effort on generating customized visualizations.
+
+3. Business Executives: The set of provided visualizations are beneficial for raising awareness among those involved in developing AI applications, allow them to audit model predictions for potential unfairness, and establish a strong governance framework around the use of AI applications.
+
+# <a name="assessment"></a>
+# Fairness Assessment Dashboard 
+The assessment dashboard automatically analyzes a model’s predictions, provides user with a set of insights into how the model is treating different buckets (e.g., female, male, other gender) of a protected attribute/feature (e.g., gender).  
+
+
+
+These insights are calculated via a set of disparity metrics that showcase:
+
+1) [Disparity in Accuracy]: How the model accuracy differs across different buckets of a selected protected attribute (e.g., how accuracy of the model differs for "females" vs. "males" vs. "other gender" data points),
+
+2_ [Disparity in Predictions] How the model predictions differ across different buckets of a selected protected attribute  (e.g., how many "females" have received prediction `Approved` on their loan application in contrast to "males" and "other gender" data points?). 
+
+The toolkit surfaces a “wizard” flow that allows you to set up accuracy and disparity metrics and then view them alongside various visualizations on a dashboard. 
+
+After loading the visualizations via the following API call, you will see the following steps: TBC
+
+
+
+
+
+
+
+
+# <a name="mitigation"></a>
+# Unfairness Mitigation Techniques
+
+
+
+
+
+Fairlearn includes a Python package that implements a variety of fairness-related algorithms to mitigate bias including:
+
+## Pre-processing and In-processing:
+
+Fairlearn shows that given access to a learning oracle for a class H, there is an efficient algorithm to find the lowest-error distribution over classifiers in H subject to equalizing false positive rates across polynomially many subgroups.
```

Comment:
I'm not sure that that's what fairlearn shows. fairlearn is only software. Some paper might show that. @MiroDudik is that something we may want below in algorithm descriptions as a reference to a paper?

Reply:
this is gone in Miro's version